core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review.component.html

```diff
@@ -140,19 +140,19 @@ <h3 class="oppia-mobile-question-header" title="{{questionHeader}}">
     background-color: transparent;
     border: none;
     box-sizing: border-box;
-    color: #009688;
+    color: #000000;
     font-size: 18px;
     font-weight: bold;
     min-height: 40px;
     min-width: 100px;
     text-align: center;
   }
   .oppia-pagination-button:hover {
-    border: 2px solid #009688;
+    border: 2px solid #000000;
     border-radius: 5px;
   }
   .oppia-pagination-button[disabled] {
-    color: #a9e5df;
+    color: #0000003D;
```

Comment:
Is this the hex color code? (looks like RGBA color code to me, not sure though!)

Reply:
Done